src/cocoa/toga_cocoa/widgets/base.py

```diff
@@ -11,6 +11,10 @@ def __init__(self, interface):
         self.native = None
         self.create()
         self.interface.style.reapply()
+        self.set_enabled(self.interface.enabled)
+
+    def create(self):
+        raise NotImplementedError()
```

Comment:
Is this necessary? Or have you added it for completeness?

Reply:
For completeness. Its not really related to the PR so I can take it out again if you like...